gui/src/components/tracker/TrackerSettings.tsx

```diff
@@ -59,8 +59,13 @@ export function TrackerSettingsPage() {
     if (!tracker) return;
 
     const assignreq = new AssignTrackerRequestT();
+
     assignreq.mountingRotation = QuaternionToQuatT(
-      Quaternion.fromEuler(0, +mountingOrientation, 0)
+      Quaternion.fromEuler(
+        0,
+        FixEuler(+mountingOrientation) * (Math.PI / 180),
+        0
+      )
```

Comment:
According to all the documentation, this *should* be the right order to apply and everything
```suggestion
      Quaternion.fromEuler(
        0,
        0,
        FixEuler(+mountingOrientation) * (Math.PI / 180),
        'XZY'
      )
```